modular_skyrat/modules/customization/datums/dna.dm

```diff
@@ -21,37 +71,37 @@
 	var/list/data = list()
 
 	if(features["mcolor"])
-		data += sanitize_hexcolor(features["mcolor"])
+		data += sanitize_hexcolor(features["mcolor"], include_crunch = FALSE)
 	else
-		data += random_string(DNA_BLOCK_SIZE,GLOB.hex_characters)
+		data += random_string(DNA_BLOCK_SIZE_COLOR,GLOB.hex_characters)
 	if(features["mcolor2"])
-		data += sanitize_hexcolor(features["mcolor2"])
+		data += sanitize_hexcolor(features["mcolor2"], include_crunch = FALSE)
 	else
-		data += random_string(DNA_BLOCK_SIZE,GLOB.hex_characters)
+		data += random_string(DNA_BLOCK_SIZE_COLOR,GLOB.hex_characters)
 	if(features["mcolor3"])
-		data += sanitize_hexcolor(features["mcolor3"])
+		data += sanitize_hexcolor(features["mcolor3"], include_crunch = FALSE)
 	else
-		data += random_string(DNA_BLOCK_SIZE,GLOB.hex_characters)
+		data += random_string(DNA_BLOCK_SIZE_COLOR,GLOB.hex_characters)
 	if(features["ethcolor"])
-		data += sanitize_hexcolor(features["ethcolor"])
+		data += sanitize_hexcolor(features["ethcolor"], include_crunch = FALSE)
 	else
-		data += random_string(DNA_BLOCK_SIZE,GLOB.hex_characters)
+		data += random_string(DNA_BLOCK_SIZE_COLOR,GLOB.hex_characters)
 	if(features["skin_color"])
-		data += sanitize_hexcolor(features["skin_color"])
+		data += sanitize_hexcolor(features["skin_color"], include_crunch = FALSE)
 	else
-		data += random_string(DNA_BLOCK_SIZE,GLOB.hex_characters)
+		data += random_string(DNA_BLOCK_SIZE_COLOR,GLOB.hex_characters)
```

Comment:
Add spaces between the comma after an argument and the next argument, makes everything more readable.
(Damn you GitHub for not letting me make a suggestion for this :waa:)